Examples/Tests/Macroscopic_Maxwell/inputs_3d

```diff
@@ -9,6 +9,7 @@ amr.n_cell = 64 64 512
 amr.max_grid_size = 512
 amr.blocking_factor = 64
 geometry.coord_sys   = 0
```

Comment:
```suggestion
```
It looks like with the new input, `geometry.dims` we dont need `geometry.coord_sys` anymore, since it generates a warning and is unused.
Please could you remove the `geometry.coord_sys` as well so we dont get these warning . Thank you

Reply:
I was wondering how were you able to know that. Did you run a test case with such an input and checked and see that it gives a warning or it is through CI tests (that we could discuss sometime next week, if you have time?)

Reply:
When you run an artemis simulation, after 1 time step there is a report in the standard output regarding which variables were not used.  If I run inputs_maurice, but with "geom.coord_sys=1" added back in I see:
>> mpiexec -n 4 ./Bin/main3d.gnu.TPROF.MTMPI.OMP.GPUCLOCK.ex ./Examples/Tests/circuits/inputs_maurice 
MPI initialized with 4 MPI processes
MPI initialized with thread support level 3
OMP initialized with 1 OMP threads
AMReX (21.10-170-gb7050b410812) initialized
WarpX (82bc0e3f2b71)
Level 0: dt = 2.67446263e-17 ; dx = 2e-08 ; dy = 1e-07 ; dz = 1e-08
we are in init data of macro 

Grids Summary:
  Level 0   2 grids  4800000 cells  100 % of domain
            smallest grid: 200 x 60 x 200  biggest grid: 200 x 60 x 200

  Writing plotfile diags/plt00000

STEP 1 starts ...

Unused ParmParse Variables:
  [TOP]::my_constants.sigma_0(nvals = 1)  :: [0.0]
  [TOP]::my_constants.dx(nvals = 3)  :: [Lx, /, n_cellx]
  [TOP]::my_constants.dz(nvals = 3)  :: [Lz, /, n_cellz]
  [TOP]::my_constants.tiny(nvals = 1)  :: [1.e-12]


**** WARNINGS ******************************************************************
* GLOBAL warning list  after  [ FIRST STEP ]
*
* --> [!  ] [Geometry] [raised 4 times]
*     geometry.coord_sys is ignored. Please use the new geometry.dims parameter.
*     @ Raised by: ALL
*
********************************************************************************

STEP 1 ends. TIME = 2.67446263e-17 DT = 2.67446263e-17

Reply:
Thanks Andy!